From cff6fbf555a072408c21da1e818209c9d3814dd3 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 27 Oct 2015 14:36:58 +0200
Subject: [PATCH] Makefile.am: don't hardcode libtool name when running
 introspection tools

Upstream-Status: Pending [review on oe-core list]

[Updated to apply to 1.12 and patch gstallocators as well]

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

---
 gst-libs/gst/gl/Makefile.am            | 2 +-
 gst-libs/gst/insertbin/Makefile.am     | 2 +-
 gst-libs/gst/mpegts/Makefile.am        | 2 +-
 gst-libs/gst/allocators/Makefile.am    | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

Index: gst-plugins-bad-1.12.0/gst-libs/gst/gl/Makefile.am
===================================================================
--- gst-plugins-bad-1.12.0.orig/gst-libs/gst/gl/Makefile.am
+++ gst-plugins-bad-1.12.0/gst-libs/gst/gl/Makefile.am
@@ -171,7 +171,7 @@ GstGL-@GST_API_VERSION@.gir: $(INTROSPEC
 		--include=Gst-@GST_API_VERSION@ \
 		--include=GstBase-@GST_API_VERSION@ \
 		--include=GstVideo-@GST_API_VERSION@ \
-		--libtool="$(top_builddir)/libtool" \
+		--libtool="$(LIBTOOL)" \
 		--pkg gstreamer-@GST_API_VERSION@ \
 		--pkg gstreamer-base-@GST_API_VERSION@ \
 		--pkg gstreamer-video-@GST_API_VERSION@ \
Index: gst-plugins-bad-1.12.0/gst-libs/gst/insertbin/Makefile.am
===================================================================
--- gst-plugins-bad-1.12.0.orig/gst-libs/gst/insertbin/Makefile.am
+++ gst-plugins-bad-1.12.0/gst-libs/gst/insertbin/Makefile.am
@@ -45,7 +45,7 @@ GstInsertBin-@GST_API_VERSION@.gir: $(IN
 		--library=libgstinsertbin-@GST_API_VERSION@.la \
 		--include=Gst-@GST_API_VERSION@ \
 		--include=GstBase-@GST_API_VERSION@ \
-		--libtool="$(top_builddir)/libtool" \
+		--libtool="$(LIBTOOL)" \
 		--pkg gstreamer-@GST_API_VERSION@ \
 		--pkg gstreamer-base-@GST_API_VERSION@ \
 		--pkg-export gstreamer-insertbin-@GST_API_VERSION@ \
Index: gst-plugins-bad-1.12.0/gst-libs/gst/mpegts/Makefile.am
===================================================================
--- gst-plugins-bad-1.12.0.orig/gst-libs/gst/mpegts/Makefile.am
+++ gst-plugins-bad-1.12.0/gst-libs/gst/mpegts/Makefile.am
@@ -79,7 +79,7 @@ GstMpegts-@GST_API_VERSION@.gir: $(INTRO
 		--add-include-path=`PKG_CONFIG_PATH="$(GST_PKG_CONFIG_PATH)" $(PKG_CONFIG) --variable=girdir gstreamer-video-@GST_API_VERSION@` \
 		--library=libgstmpegts-@GST_API_VERSION@.la \
 		--include=Gst-@GST_API_VERSION@ \
-		--libtool="$(top_builddir)/libtool" \
+		--libtool="$(LIBTOOL)" \
 		--pkg gstreamer-@GST_API_VERSION@ \
 		--pkg gstreamer-video-@GST_API_VERSION@ \
 		--pkg-export gstreamer-mpegts-@GST_API_VERSION@ \
Index: gst-plugins-bad-1.12.0/gst-libs/gst/allocators/Makefile.am
===================================================================
diff --git gst-plugins-bad-1.12.0.orig/gst-libs/gst/allocators/Makefile.am gst-plugins-bad-1.12.0/gst-libs/gst/allocators/Makefile.am
index e50d077..623f092 100644
--- gst-plugins-bad-1.12.0.orig/gst-libs/gst/allocators/Makefile.am
+++ gst-plugins-bad-1.12.0/gst-libs/gst/allocators/Makefile.am
@@ -37,7 +37,7 @@ GstBadAllocators-@GST_API_VERSION@.gir: $(INTROSPECTION_SCANNER) libgstbadalloca
 		--add-include-path=`PKG_CONFIG_PATH="$(GST_PKG_CONFIG_PATH)" $(PKG_CONFIG) --variable=girdir gstreamer-@GST_API_VERSION@` \
 		--library=libgstbadallocators-@GST_API_VERSION@.la \
 		--include=Gst-@GST_API_VERSION@ \
-		--libtool="$(top_builddir)/libtool" \
+		--libtool="$(LIBTOOL)" \
 		--pkg gstreamer-@GST_API_VERSION@ \
 		--pkg-export gstreamer-badallocators-@GST_API_VERSION@ \
 		--output $@ \
